cmd/aigw/run.go

```diff
@@ -127,7 +159,7 @@ func run(ctx context.Context, c cmdRun, _, stderr io.Writer) error {
 	// Give it some time to do so, otherwise the process might become an orphan. This is the limitation of the current
 	// API of func-e library that is used by Envoy Gateway to run the Envoy process.
 	// TODO: actually fix the library and the EG accordingly.
-	time.Sleep(2 * time.Second)
+	time.Sleep(5 * time.Second)
```

Comment:
this can be removed altogether once https://github.com/envoyproxy/gateway/pull/5560 is merged